src/js/view/preview.js

```diff
@@ -1,13 +1,39 @@
 define([
-    'utils/underscore'
-], function(_) {
+    'utils/underscore',
+    'utils/helpers'
+], function(_, util) {
 
     var Preview = function(_model) {
         this.model = _model;
 
-        this.model.on('change:playlistItem', this.loadImage, this);
+        this.model.on('change:playlistItem', onPlaylistItem, this);
     };
 
+    function loadOnComplete() {
+        if (this.model.get('state') === 'complete') {
+            this.loadImage(this.model, this.model.get('playlistItem'));
+        }
+    }
+
+    function onPlaylistItem(model, playlistItem) {
+        var audio = (this.model.mediaModel.get('mediaType') === 'audio');
+        var idleManualStart = (this.model.get('state') === 'idle') && (!this.model.get('autostart'));
+        var idleMobile = (this.model.get('state') === 'idle') && (util.isMobile());
```

Comment:
Be careful that preview is not loaded when the state is IDLE in-between playlist items. You probably need to have a second listener for 'change:mediaType' to deal with audio, as well as one for 'complete' which you have, and one only for 'idle start' that avoids idle in between playlist items (similar to the controlbar issue).